runtime/themes/tokyonight.toml

```diff
@@ -1,96 +1,131 @@
-# Author: Paul Graydon <p.y.graydon@gmail.com>
+# Author: Paul Graydon <untimely.creation97@proton.me>
 
-"comment" = { fg = "comment", modifiers = ["italic"] }
-"constant" = { fg = "orange" }
+attribute = { fg = "cyan" }
+comment = { fg = "comment", modifiers = ["italic"] }
+"comment.block.documentation" = { fg = "yellow" }
+constant = { fg = "orange" }
+"constant.builtin" = { fg = "aqua" }
+"constant.character" = { fg = "light-green" }
 "constant.character.escape" = { fg = "magenta" }
-"function" = { fg = "blue", modifiers = ["italic"] }
+constructor = { fg = "aqua" }
+function = { fg = "blue", modifiers = ["italic"] }
+"function.builtin" = { fg = "aqua" }
 "function.macro" = { fg = "cyan" }
-"keyword" = { fg = "cyan", modifiers = ["italic"] }
+"function.special" = { fg = "cyan" }
+keyword = { fg = "purple", modifiers = ["italic"] }
```

Comment:
Why change the normal keyword to purple?
Just a question, no particular opinion on it as of right now.

Reply:
Coming from neovim, I try to stay as close as possible to folke's colors, as I intended when I first created the helix ones. You can see this as a long due, small correction. I just didn't want to make a PR just for this and waited for a more general update

Reply:
Sounds good to me!